core/src/layers/minitrace.rs

```diff
@@ -150,19 +150,33 @@ impl<A: Accessor> LayeredAccessor for MinitraceAccessor<A> {
     }
 
     async fn read(&self, path: &str, args: OpRead) -> Result<(RpRead, Self::Reader)> {
-        let span = Span::enter_with_local_parent("read");
         self.inner
             .read(path, args)
-            .map(|v| v.map(|(rp, r)| (rp, MinitraceWrapper::new(span, r))))
+            .map(|v| {
+                v.map(|(rp, r)| {
+                    (
+                        rp,
+                        MinitraceWrapper::new(Span::enter_with_local_parent("ReadOperation"), r),
```

Comment:
Oh, my mistake.

Please use [`ReadOperation`](https://opendal.apache.org/docs/rust/opendal/raw/oio/enum.ReadOperation.html) which provides display. For example, `ReadOperation::Read.into_static()`.

Reply:
for inner op , we can use the operation wich provids display

```
impl<R: oio::BlockingPage> oio::BlockingPage for MinitraceWrapper<R> {
    fn next(&mut self) -> Result<Option<Vec<oio::Entry>>> {
        let _span = Span::enter_with_parent(PageOperation::BlockingNext.into_static(), &self.span);
        self.inner.next()
    }
}
```